drivers/storage/portworx/component/autopilot.go

```diff
@@ -265,12 +264,6 @@ func (c *autopilot) createClusterRole() error {
 					Resources: []string{"*"},
 					Verbs:     []string{"*"},
 				},
-				{
```

Comment:
If customers are running in a pre-1.26 k8s and have PSPs enabled, we will still need this. 
To keep the support for those, we should remove this only for 1.26 and above.

Reply:
Good point. Originally I thought by removing PSP entirely and specifying the rules in securitycontext at container level should be also compatible for customer running pre-1.2.6.

Then in order to only remove for 1.26 and above, can you suggest if there is any control in operator currently like checking what version of k8s cluster that customer is running on? @pure-jliao @haibinxie

Reply:
The `Initialize()` function in this file take the current k8s version as an input. You can use that and check the k8s version against the required k8s version to remove or keep the PSPs.

Eg -https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/component/pvccontroller.go#L394

Reply:
Huh, nice pointer! Thanks a lot @piyush-nimbalkar. Change is on the way